tests/aggregation/test_binary_relevance_aggregation.py

```diff
@@ -48,4 +48,4 @@ def test_binary_relevance_aggregation_on_toy_data(aggregator: BaseClassification
 def test_binary_relevance_aggregation_on_empty(aggregator: BaseClassificationAggregator) -> None:
     mb = BinaryRelevance(aggregator)
     result = mb.fit_predict(pd.DataFrame([], columns=['task', 'worker', 'label']))
-    assert_series_equal(pd.Series(dtype=float, name='agg_label'), result)
+    assert_series_equal(pd.Series(dtype=float, name='agg_label'), result, check_index_type=False)
```

Comment:
Is there a simple way to make the index types matching?

Reply:
I would able to find out how to manually create an `Index` with `empty` dtype since dtype looks like a numpy dtype and NumPy reference doesn't say anything about such a dtype. Overall, I consider checking index here ridiculous. Moreover, we've now observed that it's kind of undefined behaviour and can vary on Pandas versions.